Dockerfile

```diff
@@ -1,7 +1,7 @@
 FROM registry.centos.org/centos/centos:7
   
 ENV APP_DIR='/f8a_report'
-
+ENV PYTHONPATH=.
 WORKDIR ${APP_DIR}
```

Comment:
```suggestion
ENV PYTHONPATH=${APP_DIR}
```

Reply:
Can't replace WORKDIR with said change

The next steps are failing:
RUN pip3 install -r requirements.txt
CMD ["f8a_report/stack_report_main.py"]